.github/workflows/integrate.yaml

```diff
@@ -37,6 +37,16 @@ jobs:
     - name: Lint code
       run: tox -e lint
 
+  terraform-checks:
+    name: Terraform
+    uses: canonical/charmed-kubeflow-workflows/.github/workflows/terraform-checks.yaml@main
+    with:
+      charm-path: .
+      # The namespace is hardcoded in the upstream project
+      # So the model's name must be kubeflow
+      # See: https://github.com/kubeflow/kubeflow/issues/6136
+      model: kubeflow
```

Comment:
I don't think this should be hardcoded for the GH profiles automator charm.

Reply:
Sounds good! Pushed a commit
https://github.com/canonical/github-profiles-automator/pull/41/commits/dfe23e78be5592fbe2b1f1bafca94c7ced7098d0